x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.test.ts

```diff
@@ -0,0 +1,1026 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resetContext } from 'kea';
+
+import { CredentialsLogic } from './credentials_logic';
+import { ADMIN, PRIVATE } from '../../constants/credentials';
+
+jest.mock('../../../shared/http', () => ({
+  HttpLogic: { values: { http: { get: jest.fn(), delete: jest.fn() } } },
+}));
+import { HttpLogic } from '../../../shared/http';
+import { IApiToken, ICredentialsDetails } from '../../../../../common/types/app_search';
+import { IMeta } from '../../../../../common/types';
+
+function mount(defaults?: object) {
```

Comment:
Minor syntax request: prefer arrow syntax, and move all vars/setup if possible into the main `describe('CredentialsLogic')` block:

```js
describe('CredentialsLogic', () => {
  const DEFAULT_VALUES = { /* etc */ };

  const mount = (defaults?: object) => {
    // etc.
```

Reply:
8ba4cee7a3075bf3d95303945befc2d35a6b13a9